### DISCUSSION TITLE
更换当前的文档框架为 vitepress ？

### DISCUSSION THREAD

**Discussion by CYTMWIA** [xmake-io/xmake#3998] [Ideas]
最近我在学习xmake的使用，但阅读文档的过程略有痛苦，包括：

1. 在左侧目录选择章节之后，在右侧滚动会造成目录侧滚动条的“瞬移”，将当前阅读一节的标题突然固定在左下角。关于这个问题，我观察发现`docsify`文档本身也有相同的问题。
2. 虽然当前的 xmake 文章数量其实不多，但每个文章都是非常长的 markdown，且小标题不少，其目录在左侧展开后想要跳到下一篇文章并不容易。
3. 搜索栏太小，而且在目录栏最上方并不明显，而且由于上面第 1 点导致想要查看另一个搜索结果非常困难。

总结一下，我认为`docsify`在目录侧赋予了过多的功能，它既有文章选择、也有文章**内容**选择，甚至搜索栏也在，导致阅读体验不佳。
最后我是直接把文档仓库下载下来在 vscode 里阅读的，虽然没有那么恼人，但还是想要在网页阅读（

最近我也在阅读 vue 的文档，其体验就好多了，他们使用的是 [VitePress](https://vitepress.dev/)。
所以我大概看了一下 VitePress，虽然还在 beta 阶段，但似乎已经可用了，且以上痛点均有解决，同时还支持 i18n。

所以能否更换文档框架以提升阅读体验？
如果作者精力有限的话，请问能否接受修改文档框架或类似 PR？

**Comment by waruqi** [comment]
不改了，当初从 docute 切 docsify ，就花了大半年时间，还要各种各种修复 link 。。不折腾了，这些文档框架 或多或少都有小问题，再换没啥意义
